lib/natalie/compiler/binding_gen.rb

```diff
@@ -554,7 +554,7 @@ def generate_name
 
 gen.binding('Parser::Sexp', 'new', 'SexpValue', 'new_method', argc: :any, pass_env: true, pass_block: false, return_type: :Value)
 gen.binding('Parser::Sexp', 'inspect', 'SexpValue', 'inspect', argc: 0, pass_env: true, pass_block: false, return_type: :Value)
-gen.binding('Parser::Sexp', 'sexp_type', 'SexpValue', 'first', argc: 0, pass_env: true, pass_block: false, return_type: :Value)
+gen.binding('Parser::Sexp', 'sexp_type', 'SexpValue', 'first', argc: 0..1, pass_env: true, pass_block: false, return_type: :Value)
```

Comment:
Should this binding be changed? I could not get it to compile without this or adding some specialization to sexpvalue.
But I could not find any spec or tests to run so not sure if this broke anything

Reply:
This is correct. Since it's actually calling through to the `ArrayValue::first` underneath, the argc needs to match. :smile: